### PR TITLE
CLI usage improvements

### PR DESCRIPTION
**Pull request by GGP1**
|Related issue|
|---|
| Closes https://github.com/wazuh/wazuh/issues/26693 |

## Description

Removes unused CLI flags and improves their behavior consistency. 

### Tests

<details><summary>wazuh-server --help</summary>

```console
root@wazuh-master:/# wazuh-server --help
usage: wazuh_server.py [-h] [-V] {start,stop,status} ...

options:
  -h, --help           show this help message and exit
  -V                   Print version

subcommands:
  {start,stop,status}  Management operations.
    start              Start Wazuh server.
    stop               Stop Wazuh server.
    status             Show the Wazuh server status.
```

</details>

<details><summary>wazuh-server -V</summary>

```console
root@wazuh-master:/# wazuh-server -V

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html
```

</details>

<details><summary>wazuh-server start -r</summary>

```console
root@wazuh-master:/# wazuh-server start -r
2024/11/19 14:34:12 DEBUG: [Cluster] [Main] Removing '/run/wazuh-server/cluster/'.
2024/11/19 14:34:12 DEBUG: [Cluster] [Main] Removed '/run/wazuh-server/cluster/'.
Starting cluster in foreground (pid: 4432)
2024/11/19 14:34:12 INFO: [Cluster] [Main] Started wazuh-engined (pid: 4433)
2024/11/19 14:34:12 INFO: [Cluster] [Main] Started wazuh-apid (pid: 4434)
2024/11/19 14:34:12 INFO: [Cluster] [Main] Started wazuh-comms-apid (pid: 4435)
2024/11/19 14:34:12 INFO: [Local Server] [Main] Serving on /run/wazuh-server/socket/local-server.sock
2024/11/19 14:34:12 DEBUG: [Local Server] [Keep alive] Calculating.
2024/11/19 14:34:12 DEBUG: [Local Server] [Keep alive] Calculated.
2024/11/19 14:34:12 INFO: [Master] [Main] Serving on ('::1', 1516, 0, 0)
2024/11/19 14:34:12 DEBUG: [Master] [Keep alive] Calculating.
2024/11/19 14:34:12 DEBUG: [Master] [Keep alive] Calculated.
2024/11/19 14:34:12 INFO: [Master] [Local integrity] Starting.
2024/11/19 14:34:12 INFO: [Master] [Local integrity] Finished in 0.097s. Calculated metadata of 0 files.
2024/11/19 14:34:13 INFO: HTTPS is enabled but cannot find the private key and/or certificate. Attempting to generate them
2024/11/19 14:34:13 INFO: Starting API in foreground
2024/11/19 14:34:13 INFO: Starting API as root
2024/11/19 14:34:13 INFO: Generated private key file in /etc/wazuh-server/api/configuration/ssl/server.key
2024/11/19 14:34:13 INFO: Generated certificate file in /etc/wazuh-server/api/configuration/ssl/server.crt
2024/11/19 14:34:13 INFO: Starting API in foreground
2024/11/19 14:34:13 INFO: Starting API as root
2024/11/19 14:34:13 INFO: Checking RBAC database integrity...
2024/11/19 14:34:13 INFO: RBAC database not found. Initializing
2024/11/19 14:34:13 INFO: Listening on localhost:27000
2024/11/19 14:34:13 INFO: Starting gunicorn 22.0.0
2024/11/19 14:34:13 INFO: Listening at: https://127.0.0.1:27000 (4435)
2024/11/19 14:34:13 INFO: Using worker: uvicorn.workers.UvicornWorker
2024/11/19 14:34:13 INFO: Booting worker with pid: 4845
2024/11/19 14:34:13 INFO: Booting worker with pid: 4846
2024/11/19 14:34:13 INFO: Started server process [4435]
2024/11/19 14:34:13 INFO: Waiting for application startup.
2024/11/19 14:34:13 INFO: Application startup complete.
2024/11/19 14:34:13 INFO: Uvicorn running on unix socket /run/wazuh-server/socket/comms-api.sock (Press CTRL+C to quit)
2024/11/19 14:34:13 INFO: Booting worker with pid: 4847
2024/11/19 14:34:13 INFO: Started server process [4847]
2024/11/19 14:34:13 INFO: Waiting for application startup.
2024/11/19 14:34:13 INFO: Application startup complete.
2024/11/19 14:34:13 INFO: Booting worker with pid: 4849
2024/11/19 14:34:13 INFO: Started server process [4849]
2024/11/19 14:34:13 INFO: Waiting for application startup.
2024/11/19 14:34:13 INFO: Application startup complete.
```

</details>

<details><summary>wazuh-server start when the server is already running</summary>

```console
root@wazuh-master:/# wazuh-server start -r
The server is already running on process 5882
```

</details>

<details><summary>Show daemons execution failures</summary>

```console
root@wazuh-master:/# wazuh-server start -r
Starting cluster in foreground (pid: 13074)
--store_path: Directory does not exist: /var/lib/wazuh-server/engine/store
Run with --help for more information.
2024/11/19 15:26:26 ERROR: [Cluster] [Main] Error starting wazuh-engined: return code 105
2024/11/19 15:26:27 INFO: Starting API in foreground
2024/11/19 15:26:27 INFO: Starting API as root
2024/11/19 15:26:27 INFO: Checking RBAC database integrity...
2024/11/19 15:26:27 INFO: RBAC database not found. Initializing
2024/11/19 15:26:27 ERROR: Error during the database migration. Restoring the previous database file
2024/11/19 15:26:27 ERROR: Error details: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Error when trying to start the Wazuh API. 2012 - Error while attempting to check RBAC database integrity: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/20/e3q8).
2024/11/19 15:26:27 ERROR: [Cluster] [Main] Error starting wazuh-apid: return code 1
wazuh-comms-apid: Non existent process 13000, removing from /run/wazuh-server...
2024/11/19 15:26:28 INFO: Starting API in foreground
2024/11/19 15:26:28 INFO: Starting API as root
2024/11/19 15:26:28 INFO: Listening on localhost:27000
2024/11/19 15:26:28 INFO: Starting gunicorn 22.0.0
2024/11/19 15:26:28 INFO: Listening at: https://127.0.0.1:27000 (13088)
...
```

</details>

<details><summary>wazuh-server status</summary>

```console
root@wazuh-master:/# wazuh-server status
wazuh-server is running...
wazuh-comms-apid is running...
wazuh-apid is not running...
wazuh-engined is not running...
```

</details>

<details><summary>wazuh-server stop</summary>

```console
root@wazuh-master:/# wazuh-server stop
2024/11/19 15:27:55 INFO: [Cluster] [Main] Shutting down wazuh-comms-apid (pid: 13329)
```

> The other daemons are not shut down because they failed

</details>

<details><summary>wazuh-server start -rd</summary>

```console
root@wazuh-master:/# wazuh-server start -rd
root@wazuh-master:/# tail -f /var/log/wazuh-server/cluster.log 
2024/11/19 15:46:32 ERROR: [Cluster] [Main] Error starting wazuh-engined: return code 105
2024/11/19 15:46:33 ERROR: [Cluster] [Main] Error starting wazuh-apid: failed during the execution
2024/11/19 15:46:34 INFO: [Cluster] [Main] Started wazuh-comms-apid (pid: 16662)
2024/11/19 15:46:35 INFO: [Local Server] [Main] Serving on /run/wazuh-server/socket/local-server.sock
2024/11/19 15:46:35 INFO: [Master] [Main] Serving on ('127.0.0.1', 1516)
2024/11/19 15:46:35 INFO: [Master] [Local integrity] Starting.
2024/11/19 15:46:35 INFO: [Master] [Local integrity] Finished in 0.098s. Calculated metadata of 0 files.
```

</details>

<details><summary>wazuh-comms-apid --help</summary>

```console
root@wazuh-master:/# wazuh-comms-apid --help
usage: wazuh_comms_apid.py [-h] [-d] [-r] [-V]

options:
  -h, --help  show this help message and exit
  -d          Run as a daemon.
  -r          Run as root
  -V          Print version
```

</details>

<details><summary>wazuh-comms-apid -V</summary>

```console
root@wazuh-master:/# wazuh-comms-apid -V

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html
```

</details>

<details><summary>wazuh-comms-apid -r</summary>

```console
root@wazuh-master:/# wazuh-comms-apid -r
wazuh-comms-apid: Non existent process 13538, removing from /run/wazuh-server...
2024/11/19 15:30:05 INFO: Starting API in foreground
2024/11/19 15:30:05 INFO: Starting API as root
2024/11/19 15:30:05 INFO: Listening on localhost:27000
2024/11/19 15:30:05 INFO: Starting gunicorn 22.0.0
2024/11/19 15:30:05 INFO: Listening at: https://127.0.0.1:27000 (13767)
2024/11/19 15:30:05 INFO: Using worker: uvicorn.workers.UvicornWorker
2024/11/19 15:30:05 INFO: Booting worker with pid: 13803
2024/11/19 15:30:05 INFO: Started server process [13767]
2024/11/19 15:30:05 INFO: Waiting for application startup.
2024/11/19 15:30:05 INFO: Application startup complete.
2024/11/19 15:30:05 INFO: Uvicorn running on unix socket /run/wazuh-server/socket/comms-api.sock (Press CTRL+C to quit)
2024/11/19 15:30:05 INFO: Booting worker with pid: 13804
2024/11/19 15:30:05 INFO: Started server process [13804]
2024/11/19 15:30:05 INFO: Waiting for application startup.
2024/11/19 15:30:05 INFO: Application startup complete.
2024/11/19 15:30:05 INFO: Booting worker with pid: 13805
2024/11/19 15:30:05 INFO: Started server process [13805]
2024/11/19 15:30:05 INFO: Waiting for application startup.
2024/11/19 15:30:05 INFO: Application startup complete.
2024/11/19 15:30:05 INFO: Booting worker with pid: 13806
2024/11/19 15:30:05 INFO: Started server process [13806]
2024/11/19 15:30:05 INFO: Waiting for application startup.
2024/11/19 15:30:05 INFO: Application startup complete.
^C2024/11/19 15:30:24 INFO: Handling signal: int
2024/11/19 15:30:24 ERROR: Worker (pid:13803) was sent SIGINT!
2024/11/19 15:30:24 INFO: Shutting down
2024/11/19 15:30:24 INFO: Error while closing socket [Errno 9] Bad file descriptor
2024/11/19 15:30:24 INFO: Shutting down
2024/11/19 15:30:24 INFO: Error while closing socket [Errno 9] Bad file descriptor
2024/11/19 15:30:24 INFO: Shutting down
2024/11/19 15:30:24 INFO: Error while closing socket [Errno 9] Bad file descriptor
2024/11/19 15:30:24 INFO: Waiting for application shutdown.
2024/11/19 15:30:24 INFO: Application shutdown complete.
2024/11/19 15:30:24 INFO: Finished server process [13804]
2024/11/19 15:30:24 INFO: Worker exiting (pid: 13804)
2024/11/19 15:30:24 INFO: Waiting for application shutdown.
2024/11/19 15:30:24 INFO: Application shutdown complete.
2024/11/19 15:30:24 INFO: Finished server process [13805]
2024/11/19 15:30:24 INFO: Worker exiting (pid: 13805)
2024/11/19 15:30:24 INFO: Waiting for application shutdown.
2024/11/19 15:30:24 INFO: Application shutdown complete.
2024/11/19 15:30:24 INFO: Finished server process [13806]
2024/11/19 15:30:24 INFO: Worker exiting (pid: 13806)
2024/11/19 15:30:24 INFO: Shutting down: Master
2024/11/19 15:30:24 INFO: Shutting down
2024/11/19 15:30:24 INFO: MuxDemuxQueue (pid: 13785) - Received signal SIGTERM, shutting down
2024/11/19 15:30:24 INFO: Batcher (pid: 13787) - Received signal SIGTERM, shutting down
```

</details>

<details><summary>wazuh-comms-apid -rd</summary>

```console
root@wazuh-master:/# wazuh-comms-apid -rd
root@wazuh-master:/# tail -n 20 /var/log/wazuh-server/comms_api.log 
2024/11/19 15:30:57 INFO: Starting gunicorn 22.0.0
2024/11/19 15:30:57 INFO: Listening at: https://127.0.0.1:27000 (13974)
2024/11/19 15:30:57 INFO: Using worker: uvicorn.workers.UvicornWorker
2024/11/19 15:30:57 INFO: Booting worker with pid: 14010
2024/11/19 15:30:57 INFO: Started server process [13974]
2024/11/19 15:30:57 INFO: Waiting for application startup.
2024/11/19 15:30:57 INFO: Application startup complete.
2024/11/19 15:30:57 INFO: Uvicorn running on unix socket /run/wazuh-server/socket/comms-api.sock (Press CTRL+C to quit)
2024/11/19 15:30:57 INFO: Booting worker with pid: 14011
2024/11/19 15:30:57 INFO: Started server process [14011]
2024/11/19 15:30:57 INFO: Waiting for application startup.
2024/11/19 15:30:57 INFO: Application startup complete.
2024/11/19 15:30:57 INFO: Booting worker with pid: 14012
2024/11/19 15:30:57 INFO: Started server process [14012]
2024/11/19 15:30:57 INFO: Waiting for application startup.
2024/11/19 15:30:57 INFO: Booting worker with pid: 14013
2024/11/19 15:30:57 INFO: Application startup complete.
2024/11/19 15:30:57 INFO: Started server process [14013]
2024/11/19 15:30:57 INFO: Waiting for application startup.
2024/11/19 15:30:57 INFO: Application startup complete.
```

</details>

<details><summary>wazuh-apid --help</summary>

```console
root@wazuh-master:/# wazuh-apid --help
usage: wazuh_apid.py [-h] [-d] [-r] [-V]

options:
  -h, --help  show this help message and exit
  -d          Run as a daemon.
  -r          Run as root
  -V          Print version
```

</details>

<details><summary>wazuh-apid -V</summary>

```console
root@wazuh-master:/# wazuh-apid -V

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html
```

</details>

<details><summary>wazuh-apid -r</summary>

```console
root@wazuh-master:/# wazuh-apid -r
2024/11/19 15:33:58 INFO: Starting API in foreground
2024/11/19 15:33:58 INFO: Starting API as root
2024/11/19 15:33:58 INFO: Checking RBAC database integrity...
2024/11/19 15:33:58 INFO: RBAC database not found. Initializing
2024/11/19 15:33:58 ERROR: Error during the database migration. Restoring the previous database file
2024/11/19 15:33:58 ERROR: Error details: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/20/e3q8)
Error when trying to start the Wazuh API. 2012 - Error while attempting to check RBAC database integrity: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/20/e3q8).
```

> The error is caused by the engine that is deleting the folder where we store RBAC

</details>

<details><summary>wazuh-apid -rd</summary>

```console
root@wazuh-master:/# tail -n 6 /var/log/wazuh-server/api.log 
2024/11/19 15:34:36 INFO: Starting API as root
2024/11/19 15:34:36 INFO: Checking RBAC database integrity...
2024/11/19 15:34:36 INFO: RBAC database not found. Initializing
2024/11/19 15:34:36 ERROR: Error during the database migration. Restoring the previous database file
2024/11/19 15:34:36 ERROR: Error details: (sqlite3.OperationalError) unable to open database file
(Background on this error at: https://sqlalche.me/e/20/e3q8)
```

> The error is caused by the engine that is deleting the folder where we store RBAC

</details>

### CODE REVIEW

**Review by javiersanchz — APPROVED** [review]
## Review

The implemented code looks good.

The CLI was tested in a testing environment and is correct.

From my side, LGTM!

File: api/scripts/wazuh_apid.py [hunk 1]
```diff
@@ -260,65 +280,34 @@ def error(self, msg, *args, **kws):
...
+def get_script_arguments() -> Namespace:
+    """Get script arguments.
+
+    Returns
+    -------
+    argparse.Namespace
+        Arguments passed to the script.
+    """
+    parser = ArgumentParser()
+    parser.add_argument('-d', action='store_true', dest='daemon', help='Run as a daemon.')
```
Comment (fdalmaup) [inline api/scripts/wazuh_apid.py:292]:
```suggestion
    parser.add_argument('-d', '--daemon', action='store_true', dest='daemon', help='Run as a daemon')
```

File: api/scripts/wazuh_apid.py [hunk 2]
```diff
@@ -260,65 +280,34 @@ def error(self, msg, *args, **kws):
...
+    parser.add_argument('-r', action='store_true', dest='root', help='Run as root')
+    parser.add_argument('-V', action='store_true', dest='version', help='Print version')
```
Comment (fdalmaup) [inline api/scripts/wazuh_apid.py:294]:
```suggestion
    parser.add_argument('-V', '--version', action='store_true', dest='version', help='Print version')
```

File: api/scripts/wazuh_apid.py [hunk 3]
```diff
@@ -260,65 +280,34 @@ def error(self, msg, *args, **kws):
...
+    """Get script arguments.
+
+    Returns
+    -------
+    argparse.Namespace
+        Arguments passed to the script.
+    """
+    parser = ArgumentParser()
+    parser.add_argument('-d', action='store_true', dest='daemon', help='Run as a daemon.')
+    parser.add_argument('-r', action='store_true', dest='root', help='Run as root')
```
Comment (fdalmaup) [inline api/scripts/wazuh_apid.py:293]:
```suggestion
    parser.add_argument('-r', '--root', action='store_true', dest='root', help='Run as root')
```
Nevertheless, we should analyze why we would need to run the API or any other service as root.

File: apis/comms_api/scripts/wazuh_comms_apid.py [hunk 4]
```diff
@@ -206,11 +205,9 @@ def get_script_arguments() -> Namespace:
         Arguments passed to the script.
     """
     parser = ArgumentParser()
-    parser.add_argument('--host', type=str, default='0.0.0.0', help='API host.')
-    parser.add_argument('-p', '--port', type=int, default=27000, help='API port.')
-    parser.add_argument('-f', action='store_true', dest='foreground', help='Run API in foreground mode.')
+    parser.add_argument('-d', action='store_true', dest='daemon', help='Run as a daemon.')
     parser.add_argument('-r', action='store_true', dest='root', help='Run as root')
-
+    parser.add_argument('-V', action='store_true', dest='version', help='Print version')
```
Comment (fdalmaup) [inline apis/comms_api/scripts/wazuh_comms_apid.py:210]:
```suggestion
    parser.add_argument('-V', '--version', action='store_true', dest='version', help='Print version')
```

File: apis/comms_api/scripts/wazuh_comms_apid.py [hunk 5]
```diff
@@ -206,11 +205,9 @@ def get_script_arguments() -> Namespace:
         Arguments passed to the script.
     """
     parser = ArgumentParser()
-    parser.add_argument('--host', type=str, default='0.0.0.0', help='API host.')
-    parser.add_argument('-p', '--port', type=int, default=27000, help='API port.')
-    parser.add_argument('-f', action='store_true', dest='foreground', help='Run API in foreground mode.')
+    parser.add_argument('-d', action='store_true', dest='daemon', help='Run as a daemon.')
```
Comment (fdalmaup) [inline apis/comms_api/scripts/wazuh_comms_apid.py:208]:
```suggestion
    parser.add_argument('-d', '--daemon',action='store_true', dest='daemon', help='Run as a daemon')
```

File: framework/scripts/wazuh_server.py [hunk 6]
```diff
@@ -319,29 +316,33 @@ def get_script_arguments() -> argparse.Namespace:
     # implemented in worker nodes.
     start_parser.add_argument('--file', help=argparse.SUPPRESS, type=str, dest='send_file')
     ####################################################################################################################
-    start_parser.add_argument('-f', help='Run in foreground', action='store_true', dest='foreground')
+    start_parser.add_argument('-d', help='Run in daemon', action='store_true', dest='daemon')
```
Comment (fdalmaup) [inline framework/scripts/wazuh_server.py:319]:
```suggestion
    start_parser.add_argument('-d', '--daemon',  help='Run as a daemon', action='store_true', dest='daemon')
```

File: framework/scripts/wazuh_server.py [hunk 7]
```diff
@@ -319,29 +316,33 @@ def get_script_arguments() -> argparse.Namespace:
...
     start_parser.add_argument('-r', help='Run as root', action='store_true', dest='root')
```
Comment (fdalmaup) [inline framework/scripts/wazuh_server.py:320]:
```suggestion
    start_parser.add_argument('-r', '--root', help='Run as root', action='store_true', dest='root')
```

File: framework/scripts/wazuh_server.py [hunk 8]
```diff
@@ -319,29 +316,33 @@ def get_script_arguments() -> argparse.Namespace:
...
 
-    start_parser.set_defaults(func=main)
+    start_parser.set_defaults(func=start)
 
     stop_parser = subparsers.add_parser('stop', help='Stop Wazuh server.')
```
Comment (fdalmaup) [inline framework/scripts/wazuh_server.py:324]:
```suggestion
    stop_parser = subparsers.add_parser('stop', help='Stop Wazuh server')
```

File: framework/scripts/wazuh_server.py [hunk 9]
```diff
@@ -319,29 +316,33 @@ def get_script_arguments() -> argparse.Namespace:
...
-    stop_parser.set_defaults(func=stop, foreground=True)
+    stop_parser.set_defaults(func=stop)
 
     status_parser = subparsers.add_parser('status', help='Show the Wazuh server status.')
```
Comment (fdalmaup) [inline framework/scripts/wazuh_server.py:327]:
```suggestion
    status_parser = subparsers.add_parser('status', help='Show the Wazuh server status')
```

File: framework/scripts/wazuh_server.py [hunk 10]
```diff
@@ -289,16 +294,8 @@ def get_script_arguments() -> argparse.Namespace:
     argparse.Namespace
         Arguments passed to the script.
     """
-
     parser = argparse.ArgumentParser()
     parser.add_argument('-V', help='Print version', action='store_true', dest='version')
```
Comment (fdalmaup) [inline framework/scripts/wazuh_server.py:298]:
```suggestion
    parser.add_argument('-V', '--version', help='Print version', action='store_true', dest='version')
```

**Review by fdalmaup — CHANGES_REQUESTED** [review]

**Comment by fdalmaup** [comment]
Also, please review the existent help messages of the not modified commands to follow the same format as the rest, i.e. a help message without the ending dot

File: api/scripts/wazuh_apid.py (hunk 3 above)
Comment (GGP1) [reply api/scripts/wazuh_apid.py:293]: I agree, but given it's currently required to run the services I didn't remove it

**Comment by GGP1** [comment]
I leave proof of the help messages after the latest changes and tests that the long flags work as expected.

<details><summary>Help messages</summary>

```console
root@wazuh-master:/# wazuh-server --help
usage: wazuh_server.py [-h] [-V] {start,stop,status} ...

options:
  -h, --help           show this help message and exit
  -V, --version        Print version

subcommands:
  {start,stop,status}  Management operations
    start              Start Wazuh server
    stop               Stop Wazuh server
    status             Show the Wazuh server status
root@wazuh-master:/# wazuh-server start --help
usage: wazuh_server.py start [-h] [-d] [-r]

options:
  -h, --help    show this help message and exit
  -d, --daemon  Run as a daemon
  -r, --root    Run as root
root@wazuh-master:/# wazuh-server stop --help
usage: wazuh_server.py stop [-h]

options:
  -h, --help  show this help message and exit
root@wazuh-master:/# wazuh-server status --help
usage: wazuh_server.py status [-h]

options:
  -h, --help  show this help message and exit
root@wazuh-master:/# wazuh-apid --help
usage: wazuh_apid.py [-h] [-d] [-r] [-V]

options:
  -h, --help     show this help message and exit
  -d, --daemon   Run as a daemon
  -r, --root     Run as root
  -V, --version  Print version
root@wazuh-master:/# wazuh-comms-apid --help
usage: wazuh_comms_apid.py [-h] [-d] [-r] [-V]

options:
  -h, --help     show this help message and exit
  -d, --daemon   Run as a daemon
  -r, --root     Run as root
  -V, --version  Print version
```

</details>

<details><summary>Long flags test</summary>

```console
root@wazuh-master:/# wazuh-server --version

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html

root@wazuh-master:/# wazuh-apid --version

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html

root@wazuh-master:/# wazuh-comms-apid --version

Wazuh 5.0.0 - Wazuh Inc

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License (version 2) as 
published by the Free Software Foundation. For more details, go to 
https://www.gnu.org/licenses/gpl.html

```

</details>

**Review by fdalmaup — CHANGES_REQUESTED** [review]
The ETA was updated to Nov 27 due to the last required changes.

**Review by fdalmaup — APPROVED** [review]
LGTM :clap:

**Pull request merged**